tests/test_kernels/pie/Cargo.toml

```diff
@@ -0,0 +1,10 @@
+[package]
+name = "test_kernel_pie"
+version = "0.1.0"
+authors = ["Tom Dohrmann <erbse.13@gmx.de>"]
+edition = "2018"
+
+[dependencies]
+bootloader = { path = "../../.." }
+x86_64 = { version = "0.13.2", default-features = false, features = ["instructions", "inline_asm"] }
```

Comment:
I recently updated the x86_64 dependency across all tests in https://github.com/rust-osdev/bootloader/pull/209. Could you update this to version `0.14.7` as well?